src/main/perl/ComponentProxy.pm

```diff
@@ -367,6 +367,33 @@ sub _setDependencies
 }
 
 
+=item _version_check
+
+Apply version related checks.
+
+Return C<SUCCESS> if there are no version-related issues;
+report an error and return undef otherwsie.
```

Comment:
yes, i put that in the comment below. but i could add it here, eg. `Current version only reports possible different versions, and always returns C<SUCCESS> (but behaviour might change in future version)`